uproot/write/compress.py

```diff
@@ -53,7 +53,7 @@ class LZ4(Compression): pass
         uproot.const.kLZMA: LZMA,
         uproot.const.kLZ4: LZ4}
 
-def write(context, cursor, givenbytes, compression, key, keycursor):
+def write(context, cursor, givenbytes, compression, key, keycursor, awkward=False):
```

Comment:
An argument named "awkward" is elsewhere used to point to a choice of awkward library (e.g. awkward-numba, awkward-cpp, ...). Here, it's a boolean flag meaning that the array is jagged (not awkward in general). How about "isjagged"?